shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/job/progress/JobProgressTest.java

```diff
@@ -68,4 +68,16 @@ public void assertGetInventoryPosition() {
         assertTrue(jobProgress.getInventoryPosition("ds1").get("ds1.t_1") instanceof PlaceholderPosition);
         assertTrue(jobProgress.getInventoryPosition("ds1").get("ds1.t_2") instanceof PrimaryKeyPosition);
     }
+    
+    @Test
+    public void assertGetInventoryFinishedPercentage() {
+        JobProgress jobProgress = getJobProgress(ResourceUtil.readFileAndIgnoreComments("job-progress.yaml"));
+        assertThat(jobProgress.getInventoryFinishedPercentage(), is(50));
+    }
+    
+    @Test
+    public void assertGetIncrementalLatestActiveTimeMillis() {
+        JobProgress jobProgress = getJobProgress(ResourceUtil.readFileAndIgnoreComments("job-progress.yaml"));
+        assertThat(jobProgress.getIncrementalLatestActiveTimeMillis(), is(0L));
+    }
```

Comment:
Could we test more cases, e.g. there's no `incremental` data, or `latestActiveTimeMillis` has positive value.